driver-core/src/test/java/com/datastax/driver/core/DnsEndpointTests.java

```diff
@@ -0,0 +1,105 @@
+package com.datastax.driver.core;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.testng.Assert.assertTrue;
+
+import java.net.InetSocketAddress;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.annotations.Test;
+
+public class DnsEndpointTests {
+
+  private static final Logger logger = LoggerFactory.getLogger(DnsEndpointTests.class);
+
+  @Test(groups = "long")
+  public void replace_cluster_test() {
+    // Configure host resolution
+    MappedHostResolverProvider.addResolverEntry("control.reconnect.test", "127.1.1.1");
```

Comment:
Could you please implement same logic as we have in `4.x`, i.e. `ccmBridge.getNodeIpAddress`

Reply:
I could but it has no use here. I would not be able to call it here because first i set up the resolver and start the cluster later.

Reply:
Yes, but why don't you change an order ?

Reply:
Let's move it out to a separate issue - https://github.com/scylladb/java-driver/issues/376